modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Twin/cli/Program.cs

```diff
@@ -169,7 +169,7 @@ private enum Op {
                             Console.WriteLine($"Using <deviceId> '{deviceId}'");
                         }
                         if (moduleId == null) {
-                            moduleId = "opctwin";
+                            moduleId = "twin";
```

Comment:
I thought we agreed to move back to the name opctwin and opcpublisher?

Reply:
We cannot without breaking backwards compatibility as we have found during testing.   Old identities named twin and publisher will now hang around in the UX and look disconnected.   Implementing migration is too costly at this point.

Reply:
@Marc, that would kind-of lead to confusion in the setup of the edge device modules, as it happened for me. One could easily end-up with both twin and opctwin and publisher and opcpublisher in the same edge device

Reply:
Can we change the UX, then? Until the last release, we had the old names anyway...

Reply:
@barnstee we can do decide separately and change UX in a separate PR, this change was just naming consistency alignment with the existing names.

Reply:
Unfortunately we wont be able to change the ux in Azure portal - if that was needed we would need to have a migration step during startup.   We should not show these "identity ids" in our ux anyway, there should be a name that can be changed by the user and that does not affect the id.